File: 1. Beginner/1008/1008.scala
object Main {
    def main(args: Array[String]) {
        val n = io.StdIn.readLine().toInt
        val m = io.StdIn.readLine().toInt
        val s = scala.io.StdIn.readDouble()
        
        println(f"NUMBER = $n\nSALARY = U$$ ${s * m}%.2f")
    }
}

File: 1. Beginner/1008/1008.r
input <- file('stdin', 'r')

n <- as.integer(readLines(input, n=1))
m <- as.integer(readLines(input, n=1))
s <- as.double(readLines(input, n=1))

cat(sprintf("NUMBER = %d\nSALARY = U$ %.2f\n", n, m * s))